pkg/cli/environment.go

```diff
@@ -111,6 +116,7 @@ func New() *EnvSettings {
 		QPS:                       envFloat32Or("HELM_QPS", defaultQPS),
 	}
 	env.Debug, _ = strconv.ParseBool(os.Getenv("HELM_DEBUG"))
+	env.ThreeWayMergeForUnstructured, _ = strconv.ParseBool(os.Getenv("HELM_THREE_WAY_MERGE_FOR_UNSTRUCTURED"))
```

Comment:
This will affect custom resources and custom resource definitions. What do you think about `ThreeWayMergeForCustomResources`, or if shorter is preferred but still properly understood, `ThreeWayMergeForCRs`?